app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
   validates_presence_of :role
 
   has_secure_password
-  
+
+  has_many :posts
```

Comment:
@jklich151 I added this has_many relationship because the test for the user model suggested it. Let me know if this needs to be revised.